.changesets/maint_garypen_m1_changes.md

```diff
@@ -0,0 +1,13 @@
+### update macOS builds in CI to use new m1 machines ([Issue #3902](https://github.com/apollographql/router/issues/3902))
```

Comment:
For this user-facing heading, I think this title should highlight the outcome/impact more than the technical implementation detail (based on title, this sounds like it is just a CI change):

```suggestion
### Pre-built binaries are only available for Apple Silicon ([Issue #3902](https://github.com/apollographql/router/issues/3902))
```